aw-datastore/src/worker.rs

```diff
@@ -69,6 +74,17 @@ struct DatastoreWorker {
     quit: bool
 }
 
+fn _unwrap_response(receiver: ResponseReceiver<Result<Responses, DatastoreError>>)
```

Comment:
Regarding this function I think either we skip this and just continue copy-pasting as we did before or we make a general-purpose solution for more responses than just empty otherwise things get inconsistent. If we make a general purpose solution it might be necessary for it to be a macro though which gets more complicated.

Reply:
Hm I personally find that just having this helper function did make my code more readable. I experimented a bit with a general purpose solution using Rust's <> generics but had problems making it work. Would probably require at least changing the `Responses` from an enum to a trait or something like that. 

Won't probably be getting that ready for this PR, so I guess I'll remove this to make things more consistent?

Reply:
It's possible to do it with a macro, I can fix that later. If we consider it temporary it doesn't really matter if you keep it as is for now.